apps/docs/packages/saml/configuration.md

```
@@ -0,0 +1,296 @@
+
+**Bundled Size:** ~305 KB minified (82 KB gzipped)
+**Additional Dependencies:**
+
+- xml-crypto (348 KB source)
+- @xmldom/xmldom (208 KB source)
+- xpath (264 KB source)
+- @xmldom/is-dom-node (32 KB source)
+
+### Bundle Size Comparison
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)